src/java.sql/share/classes/java/sql/Connection.java

```diff
@@ -1676,5 +1677,277 @@ default void setShardingKey(ShardingKey shardingKey, ShardingKey superShardingKe
     default void setShardingKey(ShardingKey shardingKey) throws SQLException {
         throw new SQLFeatureNotSupportedException("setShardingKey not implemented");
     }
+    // JDBC 4.5
```

Comment:
Would this be information to developers if added in and \@apiNote as a function in supported by JDBC 4.5.